packages/auto-tls/test/index.spec.ts

```diff
@@ -0,0 +1,5 @@
+describe('auto-tls', () => {
+  it('should fetch a TLS certificate', async () => {
+
```

Comment:
I'm sure you are aware, but a callout here for some tests. Some I can think of: 

* invalid forgeEndpoint expectations
* certifical renewal time
* certificate loading on startup (not implemented yet)
* event emitting
* triggering fetch of cert only on certain listening maddrs

Reply:
There are tests now 😅